### DISCUSSION TITLE
Unterschied TCP und ebusctl

### DISCUSSION THREAD

**Discussion by m-ask** [john30/ebusd#748] [Q&A]
Hallo!
Ich hätte noch eine Frage was in der Umsetzung der Unterschied, zwischen den Befehlen die über TCP und denen über ebusctl ausgeführt werden, ist.
Wenn ich die Wassertemperatur über TCP abfrage (auch mit -f) wird der Wert nicht aktualisiert, über ebusctl aber schon.
Danke
m-ask

**Comment by danko3** [comment]
Unfortunately not an answer but a similar problem.
My ebus adapter is version 3.0 with an Ethernet connection (john Bayer's)
ebusctl works. Telnet with the same IP and port fails:
--
 telnet 192.168.2.114 9999
Trying 192.168.2.114...
telnet: Unable to connect to remote host: Connection refused
--
My ebusd file:
EBUSD_OPTS="--configpath=/home/pi/encon/ -d enh:192.168.2.114:9999 --httpport=9998 --mqttport=1883 --scanconfig"
But the Android app Fing does not find an open port 8888, 9999 or 9998. Nevertheless ebusctl works.
--
 ebusctl info verbose
version: ebusd 23.1.23.1
update check: OK
device: 192.168.2.114:9999, enhanced, firmware 1.1[8419].2[847f], temperature 43 °C, supply voltage 5071 mV, bus voltage 20.3 V - 20.7 V
signal: acquired
symbol rate: 6
max symbol rate: 60
min arbitration micros: 7
max arbitration micros: 34
min symbol latency: 1
max symbol latency: 11
reconnects: 0
masters: 2
messages: 76
conditional: 0
poll: 61
update: 0
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 77: master #19
address 7c: slave #19, scanned "MF=ENCON;ID=  ;SW=-;HW=-"
--


What determines which port(s) are open on the ebus adapter?  (8888 or 9999, others?)
As far as I know you can not set it in ebuspicloader, which I have not used yet.
What should I do to get telnet working?

**Comment by danko3** [comment]
Why is my comment in such big font?

**Reply by Commifreak** [reply]
Markdown. Your inserted text may have markdown elements which get interpreted as such. Therefore, always paste logs or similar inside code block:

 ` ``` `
`text goes here`
` ``` `
